nbdev_preview.py

```diff
@@ -0,0 +1,19 @@
+import inspect
+from nbdev.processors import FilterDefaults
+from nbdev.quarto import nbdev_docs
+
+monkey_patch_procs = """
+all_procs = FilterDefaults().procs()
+def new_procs(self):
+    procs = [proc for proc in all_procs if proc.__name__ != 'clean_magics']
+    return procs
+FilterDefaults.procs = new_procs
+"""
+
+if __name__ == '__main__':
+    module_file_path = inspect.getfile(FilterDefaults)
+
+    with open(module_file_path, 'a') as f:
+        f.write(monkey_patch_procs)
```

Comment:
will this not cause a bug when run two times (since you will append this code twice)?

Can you use the patch decorator to override procs instead of writing code to the file directly?

Reply:
It will not cause a bug, but i see the problem with that as i wouldn't want that the nbdev.processors.py file gets duplicate lines each time we run nbdev_preview.py.
Regarding patching and overriding procs, all the things that we do to the FilterDefaults class from within our script won't be seen by other modules that import the class FilterDefaults, as we changed our imported version and not the class globally.
so when FilterDefaults get called by nbdev_procs it will call the original class and not our modified one.
that's why i think that we don't really have an option other than writing directly to the file

Reply:
very well, then at least make the code idempotent, meaning that if that string is already appended, you won't append it twice.